README.md

```diff
@@ -511,6 +511,27 @@ These settings can be changed at any time.
 * `settings.ecs.image-pull-behavior`: The behavior used to customize the [pull image process](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html#ecs-agent-availparam) for your container instances.
   Supported values are `default`, `always`, `once`, `prefer-cached`, and the default is `default`.
 
+#### Container runtime settings
+
+Some behavior of the container runtime (currently `containerd`) can be modified with the following settings:
+
+* `settings.container-runtime.enable-unprivileged-icmp`: Allow unprivileged containers to open ICMP echo sockets.
+* `settings.container-runtime.enable-unprivileged-ports`: Allow unprivileged containers to bind to ports < 1024.
+* `settings.container-runtime.max-concurrent-downloads`: Restricts the number of concurrent downloads for each image.
+* `settings.container-runtime.max-container-log-line-size`: Controls how long container log messages can be.
+  If the log output is longer than the limit, the log message will be broken into multiple lines.
+
+Example container runtime settings:
+
+```toml
+[settings.container-runtime]
+# Set log line length to unlimited
```

Comment:
nit: I would tend to put this a little further down, near `oci-hooks` and `container-registry` settings.

Reply:
Moved.